app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -61,7 +61,19 @@ angular.module('Bastion.components').factory('Nutupane',
             params.paged = true;
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
-
+            /*
+             Start of changes: Fixes Issue #19960
+             Creating a flag "overrideAutoLoad" which will be part of params and will determine whether
+             the self loading of page needs to happen.
+             Use: Add an additional hash to the params named overrideAutoLoad and set it to true or false depending on your need.
+             If set to false, Nutupane.load (or refresh() etc..) needs to be called explicitly
+            */
+
+            self.overrideAutoLoad = params['overrideAutoLoad'] || false;
```

Comment:
The params object is sent to the API whenever a call is made. Does the API just ignore this parameter? Perhaps we should be deleting this from the params object so as not to accidentally pass it? I could see this being confusing as we are now passing configuration about Nutupane in an object designed around what should be passed to the API.